tests/test_bs.py

```diff
@@ -388,6 +390,156 @@ def test_transaction(setup_network):
+    submitapi = 'v1/submittransaction'
+    balanceapi = 'v1/balance'
+
+    accounts = { tap: { "priv": "81c90dd832e18d1cf9758254327cb3135961af6688ac9c2a8c5d71f73acc5ce5", "nonce": 0, "send": [], "recv": [] } }
+
+    def random_node():
+        rnd = random.randint(0, len(setup_network.clients.pods)-1)
+        return setup_network.clients.pods[rnd]['pod_ip'], setup_network.clients.pods[rnd]['name']
+
+    def expected_balance(acc):
+        return sum([int(tx["amount"]) for tx in accounts[acc]["recv"]]) - sum(int(int(tx["amount"]) + (int(tx["gasprice"])*tx_cost)) for tx in accounts[acc]["send"])
+
+    def random_acconut():
+        pk, info = random.choice(list(accounts.keys()))
+        return pk, info
+
+    def new_account():
+        priv, pub = genkeypair()
+        strpub = bytes.hex(pub)
+        accounts[strpub] = { "priv": priv, "nonce": 0, "recv": [], "send": [] }
+        return strpub
+
+    def transfer(frm, to, amount=None, gasprice=None, gaslimit=None):
+        pod_ip, pod_name = random_node()
+        txGen = tx_generator.TxGenerator(pub=frm, pri=accounts[frm]['priv'])
+        if amount is None:
+            amount = random.randint(1, expected_balance(frm))
+        if gasprice is None:
+            gasprice = int(amount/10)
+        if gaslimit is None:
+            gaslimit = gasprice+1
+        txBytes = txGen.generate(to, accounts[frm]['nonce'], gaslimit, gasprice, amount)
+        accounts[frm]['nonce']+=1
+        data = '{"tx":'+ str(list(txBytes)) + '}'
+        out = api_call(pod_ip, data, submitapi, testconfig['namespace'])
+        print("submit transaction from {3} to {0} of {1} with gasprice {2}".format(to, amount, gasprice, frm))
+        print(data)
+        print(out)
+        if "{'value': 'ok'}" in out:
+            accounts[to]["recv"].append({ "from": txGen.publicK, "amount": amount, "gasprice": gasprice})
+            accounts[frm]["send"].append({"to": to, "amount": amount, "gasprice": gasprice})
+            return True
+        return False
+
+    def test_account(acc, init_amount=None):
+        pod_ip, pod_name = random_node()
+        #check nonce
+        data = '{"address":"'+ tap +'"}'
+        print("checking {0} nonce".format(acc))
+        out = api_call(pod_ip, data, nonceapi, testconfig['namespace'])
+        print(out)
+        if str(accounts[acc]['nonce']) in out:
+            # check balance
+            data = '{"address":"' + str(acc) + '"}'
+            print(data)
+            out = api_call(pod_ip, data, balanceapi, testconfig['namespace'])
+            print(out)
+            balance = 0
+            if init_amount is not None:
+                balance += init_amount
+            balance = balance + expected_balance(acc)
+            if "{'value': '" + str(balance) + "'}" in out:
+                print( "{0}, balance ok ({1})".format(str(acc), out))
+                return True
+            return False
+        return False
+
+
+    # send tx to client via rpc
+    TEST_TXS = 10
```

Comment:
export this to config

Reply:
note that this is a specific test constant, I don't think that we need to export specific test-related constants to our global config. also this code is always run with the same constant config file so it won't really have an effect as if changing the value here or in config isn't much different.